src/main/java/com/google/codeu/demo/SentimentExample.java

```diff
@@ -0,0 +1,23 @@
+package com.google.codeu.demo;
+
+import com.google.cloud.language.v1.Document;
+import com.google.cloud.language.v1.Document.Type;
+import com.google.cloud.language.v1.LanguageServiceClient;
+import com.google.cloud.language.v1.Sentiment;
+import java.io.IOException;
+
+public class SentimentExample {
+  
+  public static void main(String[] args) throws IOException {
+
+    String text = "I hate coding!";
```

Comment:
Nice example lmao. However I think you should delete this file from the online repository. Its fine to keep it locally for your testing purposes, but since its not a part of the working project I don't think it should be merged to the master branch